sdk/python/kfp/containers/_build_image_api.py

```diff
@@ -79,17 +80,19 @@ def build_image_from_working_dir(image_name: str = None, working_dir: str = None
         file_filter_re: Optional. A regular expression that will be used to decide which files to include in the container building context.
         timeout: Optional. The image building timeout in seconds.
         base_image: Optional. The container image to use as the base for the new image. If not set, the Google Deep Learning Tensorflow CPU image will be used.
-        builder: Optional. An instance of ContainerBuilder or compatible class that will be used to build the image.
+        builder: Optional. An instance of :py:class:`kfp.containers.ContainerBuilder` or compatible class that will be used to build the image.
```

Comment:
Just wanted to highlight the syntax we can use to add in cross-SDK reference hyperlinks. There are a bunch more than just `:py:class:` but basically all follow a similar format: https://www.sphinx-doc.org/en/1.7/domains.html#python-roles